climakitae/core/data_load.py

```diff
@@ -1332,19 +1332,19 @@ def read_catalog_from_select(selections: "DataParameters") -> xr.DataArray:
                 )
 
         # ------ Set attributes ------
-        # Some of the derived variables may be constructed from data that comes from the same institution
-        # The dev team hasn't looked into this yet -- opportunity for future improvement
-        data_attrs = data_attrs | {"institution": "Multiple"}
-        da.attrs = data_attrs
-
-        # Convert units
+        # Convert units before copying data attributes
         da = convert_units(da, selected_units=orig_unit_selection)
         da.name = orig_variable_selection  # Set name of DataArray
 
         # Reset selections to user's original selections
         selections.variable_id = [orig_var_id_selection]
         selections.units = orig_unit_selection
 
+        # Some of the derived variables may be constructed from data that comes from the same institution
+        # The dev team hasn't looked into this yet -- opportunity for future improvement
```

Comment:
Is this something we need to look into?

Reply:
Probably low priority but it could be worth checking which derived variables are getting what attributes and if there are any that need updating at this point.

Reply:
Seems like a non-issue with `new-core` coming down the pipeline